docs/package.json

```diff
@@ -41,14 +41,15 @@
   },
   "devDependencies": {
     "@types/node": "^20",
-    "@types/react": "^18",
-    "@types/react-dom": "^18",
+    "@types/react": "^18.0.25",
+    "@types/react-dom": "^18.0.9",
     "autoprefixer": "^10.0.1",
     "eslint": "^8",
     "eslint-config-next": "14.1.0",
     "postcss": "^8",
     "prettier-plugin-tailwindcss": "^0.5.11",
     "tailwindcss": "^3.3.0",
     "typescript": "^5"
-  }
+  },
+  "includedScripts": ["build:site", "lint"]
```

Comment:
I see this is an nx option. added for "safety" ?

Reply:
Oh, I was just testing this, it allows hiding scripts from NX. Totally not needed